core/src/main/java/org/elasticsearch/search/aggregations/pipeline/bucketmetrics/stats/extended/ExtendedStatsBucketParser.java

```diff
@@ -61,4 +52,14 @@ protected ExtendedStatsBucketPipelineAggregatorBuilder buildFactory(String pipel
     public ExtendedStatsBucketPipelineAggregatorBuilder getFactoryPrototype() {
         return ExtendedStatsBucketPipelineAggregatorBuilder.PROTOTYPE;
     }
+
+    @Override
+    protected boolean token(XContentParser parser, QueryParseContext context, String field,
+                            XContentParser.Token token, Map<String, Object> params) throws IOException {
+        if (context.parseFieldMatcher().match(field, SIGMA) && token.isValue()) {
```

Comment:
Could we make the check `token == XContentParser.Token.VALUE_NUMBER` here instead of `token.isValue()` so we at least force it to be a number of some kind?